src/viam/sdk/components/camera/server.cpp

```diff
@@ -1,5 +1,8 @@
+#include <chrono>
```

Comment:
See above comment re: moving this block down.

Reply:
moved to camera.hpp